### PR TITLE
Milestone 2 Review

### PR DESCRIPTION
**Pull request by JSmith989**
## Description
We worked together to add a flights view CRUD as well as the snacks and meal menu inside food services.

## Related Issue

Milestone2:
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/30

Update Flight
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/91
Read Flight
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/96
Update - make select type selected
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/96
Add food to a menu
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/90
Show snacks button should show snacks
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/98
Show menus button should show menu
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/97
Update food - make select type selected, also add the mealtype to the food card
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/96
Delete Flight
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/92
Add Food to menu
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/90
Manu View
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/issues/73

Bug --
https://github.com/nss-evening-cohort-13/pan-am-airlines-dashboard-pan-am-airlines-dashboard/pull/105

## Motivation and Context
In order to complete week 2 of the assignment. We needed to add flights and a menu.

## How Can This Be Tested?
git fetch origin Development
git checkout Development

or 

https://fir-pan-am.web.app/

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/components/cards/flightBoardCard.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// import firebase from 'firebase/app';
```
Comment (drteresavasquez) [inline src/javascripts/components/cards/flightBoardCard.js:1]: Remove commented code

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
Remove comments and update home view, GTG!

File: src/javascripts/components/cards/flightBoardCard.js (hunk 1 above)
Comment (JSmith989) [reply src/javascripts/components/cards/flightBoardCard.js]: we removed it

**Review by drteresavasquez — APPROVED** [review]
Looks good!

**Pull request merged**